jenkins/agent-ctl.sh

```diff
@@ -75,12 +75,21 @@ while true; do
       FAKE=yes
       shift
       ;;
+    --previous)
+      # Currently jenkins-agent-20160926-0059
+      IMAGE_FLAG='--image=jenkins-agent-20160613-2240'
+      ;;
+    --previous-pr)
+      # Currently jenkins-agent-20160926-0000
+      IMAGE_FLAG='--image=jenkins-agent-20160613-1431'
+      PR=yes
+      ;;
     --pr)
       PR=yes
       shift
       ;;
     --base-image)
-      IMAGE_FLAG="--image=${BASE_IMAGE}"
+      IMAGE_FLAG="--image-family=${BASE_IMAGE}"
```

Comment:
Yes this is intentional. Updated the flag doc.

Now that image-families exist and are GA gcloud will drop support for `--image=debian-7-backports` (or `--image=debian-8`) being a special case alias for "list the current debian-7-backports and find the latest non-deprecated one."

base image in this case means the image we use to create the image (aka the `/debian-cloud/debian-8` image instead of the `jenkins-agent` image)